dev-libs/tvision/tvision-2.2.1.4.ebuild

```diff
@@ -0,0 +1,67 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Text User Interface that implements the well known CUA widgets"
+HOMEPAGE="http://tvision.sourceforge.net/"
+MY_PVR=${PVR:0:5}-${PVR:6}
+SRC_URI="mirror://sourceforge/tvision/rhtvision_${MY_PVR}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="+X debug gpm"
+
+DOCS=( readme.txt THANKS TODO )
+HTML_DOCS=( www-site/. )
+
+S=${WORKDIR}/${PN}
+
+# installed lib links to those
+RDEPEND="sys-libs/ncurses:0=
+	X? (
+		x11-libs/libX11
+		x11-libs/libXmu
+		x11-libs/libxcb
+		x11-libs/libXt
+		x11-libs/libXext
+		x11-libs/libXau
+		x11-libs/libXdmcp
+		x11-libs/libSM
+		x11-libs/libICE
+	)
+	dev-libs/libbsd
+	sys-apps/util-linux
+	gpm? ( sys-libs/gpm)"
+DEPEND="${RDEPEND}"
+
+PATCHES=(
+	"${FILESDIR}/${P}-fix-dot-INC.patch"
+	"${FILESDIR}/${P}-ldconfig.patch"
+	"${FILESDIR}/${P}-build-system.patch"
+	"${FILESDIR}/${P}-gcc6.patch"
+	"${FILESDIR}/${P}-flags.patch"
+)
+
+src_configure() {
+	./configure --fhs \
```

Comment:
can't you use econf ?

Reply:
Don't think so, it's no autoconf configure script, but an ancient perl based configure. It's copied over from the current ebuild, so I was guessing this doesn't work.

Reply:
oh then you are right :)

Reply:
Perhaps you should leave comment about it.

Reply:
Good point! I'll do it

Reply:
Added comment about configure script, please review